requirements.txt

```diff
@@ -1,3 +1,7 @@
 docopt==0.6.2
 stomp-py==8.1.2
 websocket-client==1.8.0
+argparse
```

Comment:
Thanks for keeping this updated. Can you include the versions as well? You can just run:

```
pip freeze > requirements.txt
```